playbooks/common/openshift-cluster/scaleup.yml

```diff
@@ -1,6 +0,0 @@
----
-- include: evaluate_groups.yml
-
-- include: ../openshift-node/config.yml
-  vars:
-    openshift_deployment_type: "{{ deployment_type }}"
```

Comment:
I don't think we have docs for the advanced or quick installer process. I'll double check and create an issue.